images/go/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@Kirikami why do you duplicate this config twice ? Add make command that will COPY this file some filder. Than COPY that folder files into specific "docker folder".